src/Graby.php

```diff
@@ -70,12 +72,25 @@ public function __construct($config = [], Client $client = null, ConfigBuilder $
 
         $this->config = $resolver->resolve($config);
 
-        $this->debug = (bool) $this->config['debug'];
-        $this->logger = new NullLogger();
+        $this->debug = $this->config['debug'];
+        $this->logLevel = $this->config['log_level'];
 
+        // Debug mode can be activated with 'debug' => true
+        // More details on the retrieved code and its consequent modifications
+        // can be obtained using 'log_level' = 'debug'
```

Comment:
That line can be at the end of the previous one, it won't be a _too long_ comment on one line

Reply:
https://github.com/j0k3r/graby/pull/184/commits/df26846ea7690e95bf00a445d0be095e799884eb